libmultilabel/nn/data_utils.py

```diff
@@ -221,46 +222,31 @@ def load_or_build_text_dict(
         normalize_embed (bool, optional): Whether the embeddings of each word is normalized to a unit vector. Defaults to False.
 
     Returns:
-        torchtext.vocab.Vocab: A vocab object which maps tokens to indices.
+        tuple[torchtext.vocab.Vocab, torch.Tensor]: A vocab object which maps tokens to indices and the pre-trained word vectors of shape (vocab_size, embed_dim).
     """
     if vocab_file:
         logging.info(f'Load vocab from {vocab_file}')
         with open(vocab_file, 'r') as fp:
-            vocab_list = [PAD] + [vocab.strip() for vocab in fp.readlines()]
-        vocabs = Vocab(collections.Counter(vocab_list), specials=[UNK],
-                       min_freq=1, specials_first=False)  # specials_first=False to keep PAD index 0
+            vocab_list = [[PAD] + [vocab.strip() for vocab in fp.readlines()]]
+        # special_first=False to keep PAD index 0
```

Comment:
I think it's for aligning `padding_idx` [here](https://github.com/ASUS-AICS/LibMultiLabel/blob/ebd37b3c2ce05662e83d3c86a418bc0594904192/libmultilabel/nn/networks/modules.py#L20).

Reply:
Got it, thanks!